lib/core/shared/sdk/embeddedSdk.ts

```diff
@@ -154,7 +154,7 @@ export class EmbeddedSDK extends Kuzzle {
     options: { propagate?: boolean } = {}
   ): Promise<ResponsePayload> {
     // By default, do not propagate realtime notification accross cluster nodes
-    if ( isPlainObject(request)
+    if (isPlainObject(request)
```

Comment:
This was the good indentation so every condition start on the same column
```suggestion
    if ( isPlainObject(request)
```